src/include/daos_srv/daos_server.h

```diff
@@ -186,6 +204,39 @@ struct dss_drpc_handler {
 	drpc_handler_t	handler;	/** dRPC handler for the module */
 };
 
+struct srv_obj_profile {
```

Comment:
this probably not specific for obj module? (if want to add profile to other module later). maybe srv_profile_item?

Reply:
Ah, yes, indeed.  I will update this later. still trying to figure out what information we are interested.